virtcontainers/qemu.go

```diff
@@ -613,6 +613,9 @@ func (q *qemu) virtiofsdArgs(sockPath string) []string {
 		args = append(args, "-f")
 	}
 
+	if len(q.config.VirtioFSExtraArgs) != 0 {
+		args = append(args, q.config.VirtioFSExtraArgs...)
+	}
```

Comment:
It would be good to add a test to the existing `TestQemuVirtiofsdArgs()` unit test to ensure the `-o "hello world"` type of option is handled correctly.